server/users.js

```diff
@@ -1198,6 +1213,7 @@ class User extends Chat.MessageContext {
 			// We're not resetting .trusted/.autoconfirmed so those accounts
 			// can still be locked after logout.
 		}
+		// NOTE: can't do a this.update(...) at this point because we're no longer connected.
```

Comment:
This function should maybe be renamed `markDisconnected` for clarity.

Reply:
Done